build.gradle

```diff
@@ -13,9 +13,15 @@ allprojects {
   group = 'com.salesforce.mobilesdk'
   version = '7.3.0'
   repositories {
+    mavenLocal()
```

Comment:
You may not need `mavenLocal()` since you have explicit `maven` URLs declared below.